css/style.css

```diff
@@ -177,13 +177,13 @@ background: url(https://cdn.artofproblemsolving.com/attachments/5/8/0e3aad833ef3
   margin: 0px !important;
 }
 
-.modalButton:focus {
+.modalButton:focus-within {
   background-color: white;
   opacity: 0.9;
 }
 
 .modalButton:hover {
-  background-color: white;
+  background-color: rgb(132, 243, 88);
```

Comment:
why green?

Reply:
Its just a darker green than the original light green

Reply:
👍